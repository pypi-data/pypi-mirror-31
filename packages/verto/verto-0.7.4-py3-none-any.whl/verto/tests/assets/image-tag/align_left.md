{image file-path="computer-studying-turing-test.png" alignment="left"}
