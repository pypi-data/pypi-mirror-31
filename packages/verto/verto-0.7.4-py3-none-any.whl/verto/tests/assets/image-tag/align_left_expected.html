<div>
<img class="left-align" src="{% static 'computer-studying-turing-test.png' %}" />
</div>
